Add % sign to small font, simple ahi, efficiency panel and other video/panel switch mode

TobiasBales opened the pull request:
So I have been fiddling with the playuav software.
Things I added:
% sign to small font (the rssi: 97/ really annoyed me on the small font)
simple ahi (minimosd-extra style) this should make abl happy (reported issue #26)
efficiency panel (W/km or W/m)
video/panel switch mode to switch (added to the previously implemented cycle)
Video switches >= 1500us and < 1500us, panels are distributed evenly between 950 and 2050us

I did however not add the needed flags to the config utility since it is hard to use and the code seems even worse to navigate but instead set out and made a new config utility based on githubs electron.
It is cross platform (os x and windows builds will be linked in the readme today, linux works but need to setup a vm to build it again)
It can be found at https://github.com/TobiasBales/PlayuavOSDConfigurator

TobiasBales committed:
feat: add % sign to small font and use it

Also do not show % sign for raw rssi and add % to throttle

TobiasBales committed:
feat: add simple ahi

TobiasBales committed:
feat: save some space on value labels

TobiasBales committed:
feat: add panel/video switch mode, efficiency panel

TobiasBales committed:
feat: add thr to throttle panel when no scale is shown

kwikius commented:
@TobiasBales 
My hunch is the repo isnt being maintained any more. Maybe you would like to take it over?

TobiasBales commented:
@kwikius 
Yeah I'd gladly, tried to contact him on playuav.com already but so far no response.
I was hoping for access rights to this repo but if it doesn't happen I will just use mine.

TobiasBales committed:
fix: correct imperial speed/distance multipliers

TobiasBales committed:
feat: make thr label upper case

TobiasBales committed:
style: uncrustify all the things

TobiasBales committed:
Release v0.11

TobiasBales committed:
feat: add link quality panel for openlrs/ezuhf (#2)

Huge shout out to trgFPV who actually did the work, I did some code reformating which lead to
unhappiness so I cleaned it up a bit

TobiasBales committed:
feat: add raw/percentage to link quality

TobiasBales committed:
chore(ci): add circle ci integration

TobiasBales committed:
chore(ci): add hex file to artifacts (#3)

TobiasBales committed:
docs: add build status badge to readme (#4)

TobiasBales committed:
chore(ci): copy hex file to circle ci artifacts folder (#5)

TobiasBales committed:
docs: add link to current master build to readme (#6)

trgFPV committed:
feat: moving average filter for climb rate

TobiasBales committed:
style: uncrustify src/system_stm32f4xx.c

TobiasBales committed:
style: style: uncrustify inc/osdconfig.h

TobiasBales committed:
docs: add info about running uncrustify

TobiasBales committed:
chore(ci): add uncrustify to circleci runs (#10)

trgFPV committed:
style: uncrustify osdproc.c

TobiasBales committed:
Merge pull request #9 from trgFPV/master

feat: moving average filter for climb rate

TobiasBales committed:
refactor: cleanup climb rate code (#11)

TobiasBales committed:
feat: warning if home not set

TobiasBales committed:
fix: correct osd_climb_ma indexing

TobiasBales committed:
refactor: refactor drawing into smaller functions

TobiasBales committed:
feat: home direction

TobiasBales committed:
chore(ci): use git diff with --no-pager to avoid timeouts

TobiasBales committed:
style: uncristify src/osdproc.c again

TobiasBales committed:
docs(readme): add inc/* to the uncrustify instructions

TobiasBales committed:
feat: add home latitude and longitude
